### ISSUE TITLE
Test Accuracy

### ISSUE THREAD

**Issue by rahulvigneswaran** [EMUNES/pytorch-made-class-balanced-loss#1]
Hi @EMUNES , I tried this loss on a CIFAR10-LT with resnet-32. But I can't seem to get a higher test accuracy compared to just training using a cross-entropy loss.
